### PR TITLE
Locations component 3

### PR DESCRIPTION
**Pull request by gigicupp**
@masoncpott @danielharmon Hey mates, this is my working MVP branch for the "Locations" component of the Sourcing services.
-It has a lot css changes
-Added an edge case on the locations info, in case there is no info available on the product in database.

Thank you in advance!

### CODE REVIEW

File: client/src/components/location.jsx [hunk 1]
```diff
@@ -1,46 +1,136 @@
...
+const CardContainer = Styled.div`
+  background-color: #f9f9f9;
+  font-family: 'Nunito Sans', sans-serif;
+  cursor: pointer;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 0.2fr 1.5fr 1.5fr 0.7fr;
+  grid-template-rows: 0.3fr 0.3fr 1.7fr 1.7fr;
+  gap: 0px 0px;
+  grid-template-areas:
```
Comment (danielharmon) [inline client/src/components/location.jsx:24]: im jealous you dove into grid!

File: client/src/components/location.jsx [hunk 2]
```diff
@@ -1,46 +1,136 @@
...
+          <Company>Tater Tots Co., Ltd.</Company>
+          <Country>Hack Reactor</Country>
+          <CardInfo>
+            <FactMill>Factory</FactMill>
+            <LearnMore>Learn More</LearnMore>
+          </CardInfo>
+        </CardContainer>
+      </Carousel>
+    }
+    let pics = locations.length > 2
```
Comment (danielharmon) [inline client/src/components/location.jsx]: i think this should be DRYer by just mapping once here regardless of length, and slicing off three when you need them

File: client/src/components/location.jsx [hunk 3]
```diff
@@ -1,46 +1,136 @@
...
+              <LearnMore>Learn More</LearnMore>
+            </CardInfo>
+          </CardContainer>
+        })
+   
     return (
       locations.length > 0 && (
-        <div className='carousel'>
+        <Carousel>
           {pics}
```
Comment (danielharmon) [inline client/src/components/location.jsx]: {pics.slice(0,2)}

File: client/src/components/locations.jsx [hunk 4]
```diff
@@ -69,21 +86,21 @@ class Locations extends React.Component {
 
   render() {
     return (
-      <div>
-        <div className='container'>
-          <SliderButton onClick={this.slideLeft}>
+      <SliderContainer >
+        <SliderCardContainer>
+          {/* <SliderButton onClick={this.slideLeft}>
```
Comment (danielharmon) [inline client/src/components/locations.jsx]: I think i have also done this in my PRs but i doubt having commented out code in master is something you want

**Review by danielharmon — CHANGES_REQUESTED** [review]
left a few inline comments you may want to address

File: client/src/components/location.jsx (hunk 2 above)
Comment (gigicupp) [reply client/src/components/location.jsx]: @danielharmon Ha!! Didn't think of it. Will make a change.

File: client/src/components/locations.jsx (hunk 4 above)
Comment (gigicupp) [reply client/src/components/locations.jsx]: @danielharmon I need it eventually. It was on the locations tab then it got moved to materials. I haven't made the call to the db. so when that happens, these commented out codes will be needed

**Pull request merged**